test/spec/spec.controller.js

```diff
@@ -20,6 +20,11 @@ describe('lib/base-controller', () => {
     sinon.stub(BaseController.prototype, 'locals').returns({foo: 'bar'});
   });
 
+  it('sets the correct error class to the instance', () => {
+    controller = new Controller({});
+    controller.Error.should.be.equal(require('../../lib/error'));
```

Comment:
It's less important here than the case above since it's a test run, but I find "never `require` inside a function call` to be a good rule to prevent unexpected errors at runtime since it means the entire dependency tree is immediately resolved on startup.